test/AioLogger.test.js

```diff
@@ -166,3 +166,30 @@ test('with Debug and level being silly', () => {
   aioLogger.silly('message')
   expect(global.console.log).toHaveBeenCalledTimes(2)
 })
+
+test('debug with string substitution', () => {
+  process.env.AIO_LOG_LEVEL = 'debug'
+  const aioLogger = AioLogger('App', { provider: 'debug' })
+  aioLogger.info('message %s %s', 'hello', 'world')
+  expect(global.console.log).toHaveBeenLastCalledWith(
+    expect.stringContaining('message hello world')
+  )
+})
+
+test('winston debug with string substitution', async () => {
+  fs.removeSync('./logfile.txt')
```

Comment:
During testing we should mock the file system, but that was already done like this and not part of your PR, we could raise an issue for that :).